api/tests/routes/backoffice_v3/finance_test.py

```diff
@@ -315,8 +315,11 @@ def test_get_incident_history(self, legit_user, authenticated_client):
 
         rows = html_parser.extract_table_rows(response.data)
         assert len(rows) == 2
-        assert rows[0]["Type"] == history_models.ActionType.FINANCE_INCIDENT_CANCELLED.value
-        assert rows[1]["Type"] == history_models.ActionType.FINANCE_INCIDENT_CREATED.value
-        assert rows[1]["Date/Heure"].startswith(action.actionDate.strftime("Le %d/%m/%Y à "))
-        assert rows[1]["Commentaire"] == action.comment
-        assert rows[1]["Auteur"] == action.authorUser.full_name
+        iterator_actions = iter(rows)
+        cancel_action = next(iterator_actions)
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Ici ça ne résoud pas le problème d'ordre
Il faut soit ordoner, soit ajouter une condition au `next` pour être sur de quel item tu récupères

Reply:
Le truc c'est qu'on veut justement tester que les actions sont ordonnées par date, du coup ajouter une condition ou ordonner ici ca rendrait cette partie pas si utile :thinking:

Reply:
Sinon je fais en sorte d'avoir une seule ligne, on testera pas l'ordre d'affichage mais le test passera

Reply:
Ah ok si tu veux tester l'ordre ça doit être bon alors :+1: 
Bizare que ça ait cassé sur master précédement